test/uk/gov/hmrc/securemessage/services/utils/GenerateRandom.scala

```diff
@@ -16,14 +16,14 @@
 
 package uk.gov.hmrc.securemessage.services.utils
 
-import uk.gov.hmrc.domain.{ Generator, SaUtrGenerator }
+import uk.gov.hmrc.domain.{ NinoGenerator, SaUtrGenerator }
 
 import java.util.UUID
 import scala.util.Random
 
 object GenerateRandom {
   val rand = new Random()
-  val ninoGenerator = new Generator(rand)
+  val ninoGenerator = new NinoGenerator(rand)
```

Comment:
Better to use NinoGenerator() as now it has Random with its default instantiation. Resolve the comment if you wish to continue with the new instantiation